bindata/network/multus/multus.yaml

```diff
@@ -492,8 +492,8 @@ spec:
       backoffLimit: 0
       template:
         spec:
-          priorityClassName: "system-cluster-critical"
```

Comment:
Shouldn't we always specify the `priorityClassName` ?

Reply:
Good catch Miguel. I reverted this.